packages/tabs/src/LionTabs.js

```diff
@@ -108,11 +112,14 @@ export class LionTabs extends LitElement {
 
   constructor() {
     super();
+    /**
+     * An index number of the selected tab
+     */
```

Comment:
Can you explain why we need to put it here (initialisation phase) and not in the properties list (define phase)?

Reply:
typescript only knows about "javascript" and has no knowledge about web components/lit-element properties... if you want that info in the autocomplete then we will need to be put it with the js property

Reply:
I was hoping it would pick 
```js
    /**
     * An index number of the selected tab
     * @property {Number} selectedIndex
     */
```
but it doesn't...